docs/compilers/osv.md

```diff
@@ -16,3 +19,86 @@ Compiling Java on the OSv platform requires the following parameters be met:
 * Either:
   * Project packaged as a fat `.jar` file or `.war` file *or*
   * Project uses **Gradle** or **Maven** and able to be built as a fat `.jar` or `.war`
+
+---
+
+### Node.js (v4.4.5)
+Compiling Nodejs applications on OSv requires the following parameters be met:
+* One "main" file somewhere in your project
+* All dependencies already installed to `node_modules` with `npm install` (use node v4.4.5)
+* A configuration file named `manifest.yaml` in the root directory of your project.
+  * the `manifest.yaml` file should contain path to the "main" file and, optionally, logical image
+  size for the unikernel:
+    ```yaml
+    main_file: YOUR_MAIN_FILE.js
+    image_size: "10GB"  # optional, defaults to "10GB"
+    ```
+    where you replace `YOUR_MAIN_FILE.js` with the relative path to your main file from the root directory of your project.
```

Comment:
@miha-plesko If I understand this correctly, this is actually exactly the same as if you were running node app in other unikernels that are already supported in UniK? If so, please mention this, as it is a readme.